### PR TITLE
chore: flutter3.10

### PR DESCRIPTION
**Pull request by atrope**
Fix some issues for flutter 3.10

### CODE REVIEW

File: android/local.properties
Comment (dsolonenko) [inline android/local.properties:1]: This file should be ignored

File: example/ios/Flutter/flutter_export_environment.sh
Comment (dsolonenko) [inline example/ios/Flutter/flutter_export_environment.sh:1]: Seems that this file should be ignored too

**Review by dsolonenko — COMMENTED** [review]
Local files should be removed?

**Comment by atrope** [comment]
they should. i will commit only the needed one

**Comment by atrope** [comment]
@dsolonenko done and also added gitignore as recommended

**Review by dsolonenko — APPROVED** [review]

**Comment by jneufeld101010** [comment]
Hello, I just wanted to check on the status of this PR. I'm utilizing this library in an app and am eager for it to be fixed. It is breaking my build. Thank you

**Review by octomato — APPROVED** [review]

**Comment by octomato** [comment]
@jneufeld101010  Thanks to use this library, I'll create a new pub release today.

**Pull request merged**